src/opnsense/mvc/app/models/OPNsense/Core/Menu/Menu.xml

```diff
@@ -49,10 +49,7 @@
             <LogFile order="80" VisibleName="Log File" url="/ui/diagnostics/log/core/pkg"/>
         </Firmware>
         <Gateways cssClass="fa fa-location-arrow fa-fw">
-            <Single order="10" url="/system_gateways.php">
-                <List url="/system_gateways.php*" visibility="hidden"/>
-                <Edit url="/system_gateways_edit.php*" visibility="hidden"/>
-            </Single>
+            <Single order="10" url="/ui/gateways"></Single>
```

Comment:
the naming on this sucks (mea culpa)... any other way to name this? this is our chance

Reply:
what about "primary"?

Reply:
"what's your primary gateways?" is probably confusing for different reasons... more like "individual" but also strange word for a single "thing" :/

Reply:
well, the alternative are the groups - which are not functional until explicitly configured for PBR, in contrast to the "primary" gateways.

Other than that, we could also take inspiration from something like the routes and name it "Configuration" - it suggests being the "main" window as well as the benefit of consistency with other submenus.